src/index.js

```diff
@@ -108,9 +108,9 @@ export default class LineClamp extends Component {
     }
 
     if (startPos === maxTextLength) {
-      return
+      return text
```

Comment:
Yes, it doesn't need to return text, a little bit confused with react-truncate-text source code